<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageConverter</class>
 <widget class="QWidget" name="ImageConverter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Converter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select an image file:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="filePath"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnBrowse">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Convert to:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="formatCombo">
       <item>
        <property name="text">
         <string>PNG</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JPEG</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JPG</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnConvert">
     <property name="text">
      <string>Convert</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
